extensions/markdown-language-features/src/util/topmostLineMonitor.ts

```diff
@@ -7,18 +7,32 @@ import * as vscode from 'vscode';
 import { Disposable } from '../util/dispose';
 import { isMarkdownFile } from './file';
 
+export interface LastScrollLocation {
+	readonly line: number;
+	readonly uri: vscode.Uri;
+}
+
 export class TopmostLineMonitor extends Disposable {
 
 	private readonly pendingUpdates = new Map<string, number>();
 	private readonly throttle = 50;
+	private previousEditorInfo = new Map<string, LastScrollLocation>();
```

Comment:
Unlike the previous commits, line numbers from static previews and text editors are both tracked in this object. I believe that they were tracked in different objects before because text editors used the reference to the editor object to find line number, but both editors now update using the same way.